### ISSUE TITLE
Answer without valid ratings - RatingQualityByDistribution

### ISSUE THREAD

**Issue by simonkorz** [coolcrowd/object-service#191]
Trying to submit a rating causes error:

``` java
19:20:33.718 [ratpack-compute-1-8] DEBUG edu.kit.ipd.crowdcontrol.workerservice.objectservice.Communication - Trying to submit rating rating_id: 43
experiment_id: 3
rating: 9
worker: 20
 for worker 20 with route /experiments/3/answers/17/rating
19:20:33.826 [ratpack-compute-1-8] ERROR edu.kit.ipd.crowdcontrol.workerservice.ErrorHandler - an internal error occurred
java.util.concurrent.CompletionException: edu.kit.ipd.crowdcontrol.workerservice.InternalServerErrorException: Cause: Object service responded with: Error! Answer without valid ratings. In class edu.kit.ipd.crowdcontrol.objectservice.quality.ratingQuality.RatingQualityByDistribution
```

**Comment by lckr** [comment]
Should be fixed with the db-update, which avoids submitting null-ratings and thus invokes this exception

**Comment by LeanderK** [comment]
push @lucaskrauss 
no, is not fixed with the db-update since you are handling the proto objects. You can't check whether the rating is set when handling the photo-objects, because a rating of 0 counts as not set.

**Comment by lckr** [comment]
The method which throws the exception doesn't use answers at all. The RatingQualityByDistribution only checks ratings. Which are aquired via answerOperations.getRatingsOfAnswer(answerRecord).

But the error arises from a different source

**Comment by LeanderK** [comment]
i can reproduce it, so it's still here

**Comment by kelunik** [comment]
Might be fixed by #201.

**Comment by lckr** [comment]
I know the source and will fix it! Dont worry

**Comment by lckr** [comment]
Fix: #202

**Comment by LeanderK** [comment]
be happy!
